metrics/xrootd_metrics.go

```diff
@@ -1815,18 +1950,7 @@ func HandleSummaryPacket(packet []byte) error {
 // The other fields are used to update the metrics accordingly.
 // The slow_ prefix fields are used to update the slow operation histograms.
 // The other fields are used to update the counters.
-func handleOSSPacket(blobs [][]byte) error {
-	updateCounter := func(new int, old int, counter prometheus.Counter) int {
-		// if the new value is less than the old value, we know that that the counter shouldnt be incremented
-		incBy := math.Max(0, float64(new-old))
-		counter.Add(incBy)
-
-		if new < old {
-			return old
-		}
-		return new
-	}
-
+func handleOSSStats(blobs [][]byte) error {
```

Comment:
Git doesn't let you comment on lines that are far from changes, but with this function name change I think you'll also need to change the debug line that says "handleOSSPacket received an OSS packet with an unrecognized event type (%s)" (line 1979 in my current view).